ADRs/ADR0006-replace-badger-kv-store [WITHDRAWN].md

```diff
@@ -0,0 +1,225 @@
+# Replace Badger KVStore <!-- omit from toc -->
```

Comment:
@deblasis Pinging you here just for your potential curiosity.

It does a good job at summarizing the key-value postgres issue we had. @dylanlott had the idea of using postgres directly but we ultimately decided a slightly different refactor.